suite-common/toast-notifications/src/types.ts

```diff
@@ -132,6 +132,10 @@ export type ToastPayload = (
           type: 'cardano-withdrawal-error';
           error: string;
       }
+    | {
+          type: 'estimated-fee-error';
```

Comment:
No need for the whole object to be introduced as a new type. You can use the object starting on line 84 and just extend the type there.

Also I see this is the same case also for `cardano-delegate-error` and `cardano-withdrawal-error`. You can move them up together with your type and remove the objects